packages/app/src/heartbeatService.ts

```diff
@@ -36,8 +36,7 @@ import {
 import { logger } from './logger';
 
 const MAX_HEADER_BYTES = 1024;
-// 30 days
-const STORED_HEARTBEAT_RETENTION_MAX_MILLIS = 30 * 24 * 60 * 60 * 1000;
+const MAX_NUM_STORED_HEARTBEATS = 30;
```

Comment:
Ah here. Could you export this and import it in the test so we don't have `30` everywhere?